src/Components/App/App.tsx

```diff
@@ -5,11 +5,11 @@ import Navbar from '../Navbar/Navbar'
 import { Bookshelf } from '../Bookshelf/Bookshelf'
 import './App.css';
 import { Switch, Route } from 'react-router-dom'
-import { State, SingleBook} from '../../types'
+import { AppState, SingleBook} from '../../util/types'
```

Comment:
I think this is better for readability and dev empathy, also I like it better in the util directory.